app/views/user_sessions/new.html.erb

```diff
@@ -40,10 +40,12 @@
       <h5>OR </h5>
       <h3> Login with </h3>
       <div class="loginbutton" style="float:left">
-        <a href="/auth/google_oauth2" id="connect-google"><span class="btn btn-default" style="margin-right:2px"><i class="fa fa-google" style="font-size:20px;color:red;"></i></span></div></a>
-        <a href="/auth/github" id="connect-github"><span class="btn btn-default" style="margin-left:2px"><i class="fa fa-github" style="font-size:20px;color:green;"></i></span></div></a>
+        <a href="/auth/google_oauth2" id="connect-google"><span class="btn btn-default" style="margin-right:2px"><i class="fa fa-google" style="font-size:20px;color:red;"></i></span></a>
+        <a href="/auth/github" id="connect-github"><span class="btn btn-default" style="margin-left:2px"><i class="fa fa-github" style="font-size:20px;color:green;"></i></span></a>
       </div>
```

Comment:
Hi @dewanhimanshu !
Can you confirm is this - `</div>` extra here ?

Reply:
this is  wrapping div

Reply:
Hi , 
You have that one in line 47 😄 !

Reply:
removed the extra div